packages/odyssey-react-mui/src/Typography.tsx

```diff
@@ -0,0 +1,285 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  Typography as MuiTypography,
+  TypographyProps as MuiTypographyProps,
+} from "@mui/material";
+import { ElementType, ReactNode } from "react";
+
+export const typographyVariantValues = {
+  h1: "h1",
+  h2: "h2",
+  h3: "h3",
+  h4: "h4",
+  h5: "h5",
+  h6: "h6",
+  body: "body1",
+  caption: "subtitle1",
```

Comment:
I'm meeting with @taylorridgway-okta to talk about how we want to handle `subtitle1` and `subtitle2`. They're both used under the hood and have horrid names, so I'd love to be able to swap in `<Caption>` and `<WhateverWeCallSubtitle2>` once this is merged.

Reply:
Mk, working names are:

`subtitle1: caption`
`subtitle1: bodyLight / body.light`

Not sure how well that translates, but the component `<Typography>`-level use is really low. We only use `subtitle1` for the "Optional" text on Field labels. We use `subtitle1` for the Form and Fieldset description text.